network-api/networkapi/buyersguide/templates/catalog.html

```diff
@@ -45,13 +45,13 @@ <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
 
         <figure class="product-box d-flex flex-column justify-content-between{% if product.draft %} draft-product{% endif %}{% if product.adult_content %} adult-content{% endif %}" data-creepiness="{{ product.votes.creepiness.average }}">
           <div class="top-left-badge-container">
-              {% include "fragments/seal_of_approval.html" with product=product %}
-              {% if product.votes.confidence %}
-                  {% if product.votes.confidence.1 > product.votes.confidence.0 %}
-                      <div class="d-none recommendation positive"></div>
-                  {% else %}
-                      <div class="d-none recommendation negative"></div>
-                  {% endif %}
+              {% if product.privacy_ding %}
+              <img
+                src="{% static "_images/buyers-guide/icon-privacy-ding-mini.svg" %}"
+                title="{% trans "*privacy not included with this product" %}"
+                alt="{% trans " *privacy not included with this product" %}"
```

Comment:
Removing this extra space will allow us to translate this once for both attributes

```suggestion
                alt="{% trans "*privacy not included with this product" %}"
```